modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -33,14 +33,16 @@ spec:
         action: Accept  <1>
         cidr: 172.210.150.1/24 <2>
         protocol: SCTP
+        tcpFlags: SYN     <3>
         direction: Ingress
         destPortRange: 80-100
         peerIP: 10.10.10.10
-        enable: true    <3>
+        enable: true    <4>
 ----
 <1> The required `action` parameter describes the action that is taken for the flow filter rule. Possible values are `Accept` or `Reject`. 
 <2> The required `cidr` parameter provides the IP address and CIDR mask for the flow filter rule and supports IPv4 and IPv6 address formats. If you want to match against any IP address, you can use `0.0.0.0/0` for IPv4 or `::/0` for IPv6.
-<3> You must set `spec.agent.ebpf.flowFilter.enable` to `true` to enable this feature.
+<3> You can use the `tcpFlags` specification to further filter flows that have a `SYN` TCP flag. For more information, see "Detecting SYN flooding using the FlowMetric API and TCP flags".
```

Comment:
I feel this cross reference to SYN flooding is not required here. I think the other way around of referencing eBPF filtering from SYN flooding is more than enough